profiles/tests/test_activity.py

```diff
@@ -1,18 +1,126 @@
 from rest_framework.test import APITestCase, APIClient
 from profiles.models import Activity
+from authentication.models import CustomUser
 
-class TestActivityList(APITestCase):
+
+class TestCategoryList(APITestCase):
 
     def setUp(self) -> None:
+        self.sale_activity = Activity.objects.create(name='sale')
         self.transport_activity = Activity.objects.create(name="transport")
-        self.sale_activity = Activity.objects.create(name="sale")
-        self.production_activity = Activity.objects.create(name="production")
-        self.education_activity = Activity.objects.create(name="education")
+        self.education_activity = Activity.objects.create(name='education')
+        self.medicine_activity = Activity.objects.create(name="medicine")
+        self.test_person_is_admin = CustomUser.objects.create_user(
+            person_email="testactivity@testadmin.com",
+            password="Testing01",
+            person_name="testactivity",
+            person_surname="admin",
+            is_active=True,
+            is_staff=True
+        )
+        self.test_person_just_user = CustomUser.objects.create_user(
+            person_email="testactivity@testuser.com",
+            password="Testing01",
+            person_name="testactivity",
+            person_surname="user",
+            is_active=True
+        )
 
     def tearDown(self) -> None:
         objects_to_delete = Activity.objects.all()
         objects_to_delete.delete()
```

Comment:
There's no need to clean up DB in the `tearDown` method since it's done automatically after each test method.

Reply:
Thank you for feedback. Done.